docs/arch/adr-03.md

```diff
@@ -0,0 +1,86 @@
+- shared access signature permissions can only be granted in account level or container level. See [here](https://docs.microsoft.com/en-us/rest/api/storageservices/create-user-delegation-sas) for details. This means that if we want to provide a link to the file, we will need to create a new blob container for the user so that the user isn't granted access to all transcription data. This may not turn out to be a real "downside" if it turns out there's no overhead involved in creating/deleting containers on demand.
+
+### Option 2: Azure File Service
+
+**Specs:**
+Target throughput for a single file share: up to 300 MiB/sec for certain regions, Up to 60 MiB/sec for all regions
+
+Pros: 
+- allows for specifying read-only or write-only permissions on folders within the share using a shared access signature (SAS): this would give us more control in how we want to organize/store the generated zip files
+- can cache Azure file shares on on-premises file servers by using Azure File Sync for quick access
+- File Services uses the SMB protocol, which is the same protocol used on file directories on Mac and Windows machines. Therefore a file share can be mapped onto a drive on your machine, which is not possible with a blob container
+- Greater potential throughput
+
+Cons:
+- Launched in Sept 2015, hasn't been around for as long as Blob Storage.
+- Smaller max size (100 TiB: 1 TiB = 2^40 bytes). We are not expecting to come close to this size, so this isn’t much of a concern
+
+## Decision
+
+While Azure File Service does offer the enticing option of granting folder specific permissions, we don't appear to have any need for the remainder of the additional functionality that comes with File Service, which makes me reluctant to want to use it. In addition, the number of articles and resources available on communicating with Blob Storage to set up file zipping is much greater than what's available for File Service. My initial analysis has not given me any reason to think that setting up user-specific containers would be problematic, but this is an open question worth thinking about for reviewers. Hypothetically, this user-specific container could be deleted after a short time window to avoid organizational clutter and unnecessary additional costs.
+
+Ultimately, my choice is to go with Blob Storage: the more basic, simple storage tool that gives us what we need and nothing more. That being said, I'd still like to keep the option of using Azure File Service on the table, in case it turns out that we *would* benefit from the additional functionality that it offers.
+
+Final questions:
+- Blob Storage doesn't have any concrete hierarchy beyond Storage Account/Blob Container - within a container, directories are virtual, demarcated by prefixes in the file name. Will this end up being problematic for us? Will it complicate file retrieval?
```

Comment:
It's up to us to organize the blob prefix in a known container with a schema. e.g. something like `/tove-exports/project/1/approved-transcription/2` that matches our application so we can reconstruct the path from our data model / record values.

File retrieval may be helped by `append blobs` vs multiple block blobs TBD. 

If that isn't useful then it's up to use to deal with multiple blob retrieval in serial / concurrent.

Reply:
This is true. I think that I had some initial apprehension/confusion about the "virtual" directory system, but after some thought, I've realized that it won't be all too different to work with after all.

As for using append blobs vs multiple block blobs, I will need to look into this and update the doc based on findings.

Reply:
@camallen, I've added information about the different blob types, and an evaluation of which type I think would be best - feel free to review when you have a moment